### PR TITLE
Add recoverAddress to IWallet #43

### LINKED ISSUE #43: Add recover method in Wallet interface
add method interface to recover address from signature and message.

### PR DESCRIPTION
**Pull request by syuhei176**
adding recoverAddress method to Wallet.

Close #43

### CODE REVIEW

File: src/wallet/interfaces/IWallet.ts [hunk 1]
```diff
@@ -1,10 +1,16 @@
...
+   * @param message
+   * @param signature
+   */
+  recoverAddress(message: Bytes, signature: Bytes): Address
   /**
    * signMessage signed a hex string message
    * @param message is hex string
    */
-  signMessage(message: string): string
+  signMessage(message: Bytes): Bytes
```
Comment (daiki44) [inline src/wallet/interfaces/IWallet.ts]: Why did you change the argument type string to Bytes?

Comment (syuhei176) [reply src/wallet/interfaces/IWallet.ts]: because string(actually it is hex string) is too general for this argument. we will use common [datatypes](https://github.com/cryptoeconomicslab/wakkanay/tree/master/src/types/Codables) as possible.

Comment (daiki44) [reply src/wallet/interfaces/IWallet.ts]: Got it! Thank you!

Comment (tkmct) [reply src/wallet/interfaces/IWallet.ts]: I'm thinking that we can add String type in Codable.
https://solidity.readthedocs.io/en/v0.5.12/abi-spec.html#types

Comment (syuhei176) [reply src/wallet/interfaces/IWallet.ts]: But in this case, `string` is hex string so `Bytes` is good I guess.

**Comment by syuhei176** [comment]
> LGTM!

please approve!
